imagedephi.py

```diff
@@ -47,21 +48,21 @@ def redact_tiff_tags(ifds, tags_to_redact):
                 # tag_info['ifds'] contains a list of lists
                 # see tifftools.read_tiff
                 for sub_ifds in tag_info['ifds']:
-                    redact_tiff_tags(sub_ifds)
+                    redact_tiff_tags(sub_ifds, tags_to_redact)
 
 
-def redact_one_image(tiff_info, output_path):
+def redact_one_image(tiff_info: Dict, output_path: Path) -> None:
     ifds = tiff_info['ifds']
     tags_to_redact = get_tags_to_redact()
     redact_tiff_tags(ifds, tags_to_redact)
     tifftools.write_tiff(tiff_info, output_path)
 
 
-def get_output_path(file_path, output_dir):
-    return output_dir/f'REDACTED_{file_path.name}'
+def get_output_path(file_path: Path, output_dir: Path) -> Path:
+    return output_dir / f'REDACTED_{file_path.name}'
 
 
-def redact_images(image_dir, output_dir):
+def redact_images(image_dir: Path, output_dir: Path) -> None:
     for child in image_dir.iterdir():
         try:
             tiff_info = tifftools.read_tiff(child)
```

Comment:
In the future, I'd like to see if we can add stronger annotations for the `tifftools` API as local stubs. I can look into this once this PR is merged.

Reply:
I opened #6 to track this.